addons/l10n_pe_website_sale/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_city_public,res_city_group_public,base_address_extended.model_res_city,base.group_public,1,0,0,0
```

Comment:
do you really need to? seems like you can add sudo everywhere it's used

Reply:
@mart-e The same rule is applied to states and countries so why not for city_id, district_id, ... ?

Reply:
Update: We can leave it as it's.